scripts/travis/before_build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-yarn global add grunt-cli
+yarn global add grunt-cli@1.2.0
```

Comment:
This is weird, npm in this case would install the provided version. Also, Yarn docs are contradicting each other in the very same page:

> `yarn add package-name@1.2.3` installs a specific version of a package from the registry.

vs:

> For example, `yarn add foo@1.2.3` would accept version `1.9.1`, but `yarn add foo@1.2.3 --exact` would only accept version `1.2.3`.

@Daniel15 could you clarify which one is right?